### PR TITLE
Add supports for VOX

### PR DESCRIPTION
**Pull request by spywhere**
- Add supports for VOX
- Fix settings cannot be read on case sensitive file system

### CODE REVIEW

File: itunes.py [hunk 1]
```diff
@@ -1,7 +1,12 @@
-import sublime, sublime_plugin, subprocess
+import sublime
+import sublime_plugin
+import subprocess
+import shlex
+
 
 STATUS_KEY = "itunes-plugin"
-SETTING_NAME = "itunes.sublime-settings"
+SETTING_NAME = "iTunes.sublime-settings"
```
Comment (airtoxin) [inline itunes.py:8]: no need to change name. 
it loads another setting file.

Comment (spywhere) [inline itunes.py:8]: Well, it's seem failed to load the settings without proper case to me.

File: itunes.py [hunk 2]
```diff
@@ -23,16 +30,21 @@ def unset_track(self, view):
...
-        if not track_data: return
+        if not track_data:
+            return
         return "▶ {artist} - {name}".format(**track_data)
 
     def get_track_data(self):
-        track_p = subprocess.Popen("osascript -l JavaScript -e 'Application(\"System Events\").applicationProcesses[\"iTunes\"](); Application(\"iTunes\").currentTrack.name();'", shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        track_script = sublime.load_resource("Packages/iTunes/scripts/track.scpt")
+        artist_script = sublime.load_resource("Packages/iTunes/scripts/artist.scpt")
+        track_p = subprocess.Popen("osascript -l JavaScript -e " + shlex.quote(track_script), shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
Comment (airtoxin) [inline itunes.py:40]: it's a very good way.
but package does not have `.no-sublime-package` file, so this package installed as zipped app.
`sublime.load_resource` works correctly in zipped ?

Comment (spywhere) [inline itunes.py:40]: Yup, don't worry about that :)

**Comment by airtoxin** [comment]
:+1:

**Pull request merged**